lib/gooddata/lcm/actions/synchronize_etls.rb

```diff
@@ -0,0 +1,127 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class SynchronizeETLs < BaseAction
+      DESCRIPTION = 'Synchronize ETLs (CC/Ruby)'
+
+      PARAMS = define_params(self) do
+        description 'Client Used for Connecting to GD'
+        param :gdc_gd_client, instance_of(Type::GdClientType), required: true
+      end
+
+      # will be updated later based on the way etl synchronization
+      RESULT_HEADER = [
+      ]
+
+      class << self
+        def call(params)
+          # Check if all required parameters were passed
+          BaseAction.check_params(PARAMS, params)
+
+          results = []
+
+          client = params.gdc_gd_client
+          development_client = params.development_client
+
+          domain_name = params.organization || params.domain
+          domain = client.domain(domain_name) || fail("Invalid domain name specified - #{domain_name}")
+
+          # we will use client_id to detect the context of etl synchronization. If we don't have client_id, it means we are in Release brick
+          has_client_ids = params
```

Comment:
Can we change the condition from `has_client_ids` to something like `target_domain_different`? Would it be more accurate?